docs/modules/ROOT/pages/components.adoc

```diff
@@ -137,21 +138,23 @@ By adding the embed attribute, `is_initialized` becomes a contract entrypoint fo
 [TIP]
 ====
 Embeddable implementations, when available in this library's components, are segregated from the internal component implementation which makes it easier to safely expose.
-Components also separate standard implementations (`snake_case`) from `camelCase`.
-This trichotomy structures the API documentation design.
+Components also separate standard (granular) implementations from {mixin} implementations.
+The API documentation design reflects these groupings.
 See {erc20-component} as an example which includes:
 
-- *Embeddable implementations*
-- *Embeddable implementations (camelCase)*
-- *Internal implementations*
+- *Embeddable Mixin Implementation*
+- *Embeddable Implementations*
+- *Internal Implementations*
+- *Events*
 ====
 
 === Mixins
 
+Mixins are impls made of a combination of smaller, more specific impls.
 While separating components into granular implementations offers flexibility,
-integrating components with many implementations can appear crowded especially when the contract will use all of the component's implementations.
-To simplify this process, applicable components offer a mixin implementation.
-Component mixins allow contracts to embed all of a component's implementations through the use of a single implementation.
+integrating components with many implementations can appear crowded especially if the contract uses all of them.
+Mixins simplify this by allowing contracts to embed groups of implementations with a single directive.
+
 Compare the following code blocks to see the benefit of using a mixin when creating an account contract:
```

Comment:
```suggestion
Compare the following code blocks to see the benefit of using a mixin when creating an account contract.
```
doesn't it look weird to use the colon and then a subheading?

Reply:
Haha indeed, it does